packages/indy-sdk-to-askar-migration/src/IndySdkToAskarMigrationUpdater.ts

```diff
@@ -63,10 +63,21 @@ export class IndySdkToAskarMigrationUpdater {
     const {
       config: { walletConfig },
     } = agent
-    if (!walletConfig) throw new IndySdkToAskarMigrationError('Wallet config is required for updating the wallet')
+    if (typeof process?.versions?.node !== 'undefined') {
+      throw new IndySdkToAskarMigrationError('Node.js is currently not officialy supported')
```

Comment:
If it's throws an error, there's no way to do it unofficial right? :)

```suggestion
      throw new IndySdkToAskarMigrationError('Node.js is currently not supported')
```